services.qmd

```diff
@@ -0,0 +1,41 @@
+---
+title: Consulting
+description: "Consulting services"
+---
+
+## Problems We Solve
+
+You should consider hiring us if:
+
+- You are hitting a plateau in getting your AI to do what you want.
+- Your LLMs need to be cheaper or faster.
+- You are overwhelmed by tools & frameworks.
+- You feel lost on what to prioritize and what experiments you should run.
+- You need outside expertise to evaluate your needs and vet potential talent.
+
+## Services
```

Comment:
Would it make sense to include a workshop that runs for a few days to teach the team the essentials of evals, RAG, recommended tools, etc. Essentially stuff that you already have in the course.

Here's a consultant that conducts a [three-day positioning workshop for 65-80k a pop](https://www.aprildunford.com/consultant).